uint8 START = 0
uint8 STOP = 1
uint8 PAUSE = 2

uint8 command
string path_name
string dir
---

string message
